### PR TITLE
Alias for install_easystats

### PR DESCRIPTION
**Pull request by zen-juen**
This is actually Dom, in an occasion to showcase some github magic

### CODE REVIEW

File: R/install_easystats.R [hunk 1]
```diff
@@ -7,6 +7,8 @@ easystats_zen <- function(){
 
 #' Install the easystats suite from github
 #'
+#' This function can be used to install all the easystats package from github, either from the master branch (the stable one) with \code{install_easystats_latest} or from the dev branch \code{install_easystats_dev}.
```
Comment (strengejacke) [inline R/install_easystats.R]: Just some minor stylistic issue, I would add parantheses to `\code{install_easystats_latest}` -> `\code{install_easystats_latest()}`, and to the second `\code{}` tag as well.

Comment (DominiqueMakowski) [reply R/install_easystats.R]: @zen-juen 👀

Comment (zen-juen) [reply R/install_easystats.R]: Just done!

Comment (DominiqueMakowski) [reply R/install_easystats.R]: A true master githubber!

**Pull request merged**